Gordon360/Services/ServiceInterfaces.cs

```diff
@@ -147,6 +147,16 @@ public interface IMembershipService
         MEMBERSHIP Delete(int id);
     }
 
+    public interface IJobsService
+    {
+        IEnumerable<StudentTimesheetsViewModel> saveShiftForUser(int studentID, int jobID, DateTime shiftStart, DateTime shiftEnd, string hoursWorked, string shiftNotes, string lastChangedBy);
```

Comment:
Is lastChangedBy parameter needed here, even though it is not used in the frontend for saveShiftForUser()?

Reply:
It is, since the stored procedure this method calls requires it as a parameter. It's passed into one of the columns in the DB table.

Reply:
Okay, so there must be code I have yet to see that happens between the frontend and this that provides a lastChangedBy

Reply:
That would be in JobsController.cs

Reply:
Oh, yes, I see it